docs/guides/gradle-plugin.md

```diff
@@ -0,0 +1,100 @@
+(gradle-plugin-page)=
+
+# Using the Nextflow Gradle plugin
+
+The [Nextflow Gradle plugin](https://github.com/nextflow-io/nextflow-plugin-gradle) simplifies plugin development by configuring default dependencies needed for Nextflow integration and defining Gradle tasks for building, testing, and publishing Nextflow plugins. The Gradle plugin is versioned and published to the [Gradle Plugin Portal](https://plugins.gradle.org/), allowing developers to manage it like any other dependency. As the plugin ecosystem evolves, the Gradle plugin will enable easier maintenance and adoption of improvements. This page introduces [Nextflow Gradle plugin](https://github.com/nextflow-io/nextflow-plugin-gradle) and how to use it.
```

Comment:
I'd say to being this documentation mentioning that nextflow provides a new `plugin create` command that simplify the creation of nextflow plugin, and introduce the Gradle plugin after that, mentioning the `nextflow plugin create` uses under the hood a gradle plugin and the [nf-plugin-template](https://github.com/nextflow-io/nf-plugin-template) project

Reply:
Ok